core/plugins/org.polarsys.capella.core.preferences/src/org/polarsys/capella/core/preferences/Activator.java

```diff
@@ -384,32 +378,7 @@ public void setPropertyStore(IResource iResource, IPreferenceStore propertiesSto
    * @param capellaCommonNavigator
    */
   public void addProjectsPropertyChangeListener(IPropertyChangeListener capellaCommonNavigator) {
-
-    IProject[] projects = ResourcesPlugin.getWorkspace().getRoot().getProjects();

```

Comment:
never occurs. 
this method is called at startup.
even if there is projects in workspace, there is no "getPropertyPreferenceStore" as it is only created when user click on Project/Properties